src/ansys/geometry/core/designer/body.py

```diff
@@ -877,10 +882,18 @@ def color(self) -> str:  # noqa: D102
                 # Fetch color from the server if it's not cached
                 color_response = self._bodies_stub.GetColor(EntityIdentifier(id=self._id))
                 if color_response.color:
-                    self._color = mcolors.to_hex(color_response.color)
+                    self._color = mcolors.to_hex(color_response.color, keep_alpha=True)
 
         return self._color
 
+    @property
+    def opacity(self) -> float:  # noqa: D102
+        opacity_hex = self._color[7:]
+        if opacity_hex == "":
+            return 1
+        else:
+            return int(opacity_hex, 16) / 255
```

Comment:
```suggestion
        return int(opacity_hex, 16) / 255 if opacity_hex else 1
```
You can simplify it into this if you want =)